### ISSUE TITLE
EFI: Simplified dashboard

### ISSUE THREAD

**Issue by sheplu** [expressjs/discussions#193]
### Status

Part: Technical

### Draft

> Provide an easy to read global dashboard solution for all contributors and members of the community showing the live state of all packages, issues, PR and some more data (changes over weeks, latest release…)
Extended scorecard
Quick visualization of the organization

**Comment by dougwilson** [comment]
I think this is already there, maybe we just need to add the additional suggestion info on it? It is the badgeboard.

**Comment by rafaell-lycan** [comment]
> There already is https://expressjs.github.io/statusboard/

This is outdated IMO.

**Comment by wesleytodd** [comment]
Yeah we just need to re enable the action which rebuilds it nightly. IIRC I turned that off a few years ago since it was unclear if anyone was using it. Feel free to open a PR to re-enable that!

**Comment by bjohansebas** [comment]
I believe this has already been achieved with the activation of the statusboard. There are still things to improve in that project, but that's a separate matter from Express.
